### ISSUE TITLE
Insufficient color contrast in several locations

### ISSUE THREAD

**Issue by valeriecodes** [DCAFEngineering/dcaf_case_management#1704]
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**
Make the app accessible.

* **What feature or behavior is this required for?**
aXe identified a few elements that do not have sufficient color contrast to meet accessibility standards. Related: https://github.com/DCAFEngineering/dcaf_case_management/issues/1691

* **How could we solve this issue? (Not knowing is okay!)**
Fix all violations of type "color contrast" detailed on the second tab of this spreadsheet: https://docs.google.com/spreadsheets/d/1qh08MwZhg6ILWrzx-rMKJK3hbNv4RI8Z4s1k0osNieE/edit#gid=986571270 (some are duplicates/can probably be fixed by changing a single style rule)

* **Anything else?**
This page has a color contrast checker that would probably come in handy: https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=webdriverjs
